src/pivotal-ui/components/tables/tables.scss

```diff
@@ -119,37 +74,41 @@ parent: table
 This is a table used to display many rows of data (it is pretty much the default table). You can use grid column classes on th and td elements. See key-value table for example.
 
 ```html_example
-<table class="table table-data">
-  <thead>
-    <tr>
-      <th>Service Instance</th>
-      <th>Service Plan</th>
-      <th>Bound Apps</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>oracle-db-sct</td>
-      <td>Oracle DB</td>
-      <td>12</td>
-    </tr>
-    <tr>
-      <td>oracle-db-sct</td>
-      <td>Oracle DB</td>
-      <td>12</td>
-    </tr>
-    <tr>
-      <td>oracle-db-sct</td>
-      <td>Oracle DB</td>
-      <td>12</td>
-    </tr>
-    <tr>
-      <td>oracle-db-sct</td>
-      <td>Oracle DB</td>
-      <td>12</td>
-    </tr>
-  </tbody>
-</table>
+<div class="panel panel-basic-alt">
+  <div class="panel-body pan">
+    <table class="table table-data man">
```

Comment:
there should be two examples in the docs I think... a regular one, and then a divided one.